test/jni.cpp

```diff
@@ -100,6 +100,41 @@ extern "C" JNIEXPORT jdouble JNICALL Java_JNI_addMix(JNIEnv*,
          + a14 + a15 + a16 + a17 + a18 + a19 + a20;
 }
 
+extern "C" JNIEXPORT jint JNICALL
+    Java_JNI_addStackBoundary2(JNIEnv* e, jclass c, jobject o1, jobject o2, jobject o3,
```

Comment:
Travis is giving errors on these because we have `-Wunused-parameters` and `-Werror` turned on.  I'm not a huge fan of that warning, but the fix here is simple: either put `UNUSED` after each parameter name (e.g. `JNIEnv* e UNUSED`) or just delete the name (just `JNIEnv*`).